sync2jira/downstream_issue.py

```diff
@@ -947,6 +954,23 @@ def _update_jira_labels(issue, labels):
     issue.update(data)
     log.info('Updated %s tag(s)' % len(_labels))
 
+def _update_github_project_fields(client, existing, issue, github_project_fields):
+    """Update a Jira issue with GitHub project item field values
+
+    :param jira.client.JIRA client: JIRA client
+    :param jira.resource.Issue existing: Existing JIRA issue
+    :param sync2jira.intermediary.Issue issue: Upstream issue
+    :param list: Fields representing GitHub project item fields in GitHub and Jira
+    """
+
+    for fieldname, values in github_project_fields.items():
+        try:
+            if fieldname == 'storypoints':
+                existing.update({values[1]: str(issue.storypoints)})
```

Comment:
Do we need to hardcode a conditional for `'storypoints'` here?

```suggestion
    for name, values in github_project_fields.items():
        try:
            ghfieldname, jirafieldname = values
            existing.update({jirafieldname: str(getattr(issue, name))})
```

Reply:
Done. b921b55434d2599f3c7462d25f84c52fde66f305